R/MsExperiment.R

```diff
@@ -228,3 +368,78 @@ setReplaceMethod("metadata", "MsExperiment",
                      x@metadata <- value
                      x
                  })
+
+#' @rdname MsExperiment
+setGeneric("linkSampleData", function(object, ...)
+    standardGeneric("linkSampleData"))
+#' @rdname MsExperiment
+#'
+#' @export
+setMethod(
```

Comment:
Does this need the be a method with a generic function? Why not simply a function?

Reply:
you're right. I made it a method because I thought I might implement for `MsExperiment` and `LinkedMsExperiment`, but no need for that anymore. I changed that in the last commit.